pos_access_right/readme/DESCRIPTION.rst

```diff
@@ -5,3 +5,8 @@ This module extends Odoo Point Of Sale features, restricting possibility to cash
 * **PoS - Change Unit Price**: The cashier can change the unit price of a product in Point Of Sale;
 * **PoS - Many Orders**: The cashier can many orders at the same time;
 * **PoS - Delete Order**: The cashier can not delete a full order;
+* **PoS - Change Quantity**: The cashier can change the quantity of a product in Point Of Sale;
+* **PoS - Delete Order Line**: The cashier can delete the order line in Point Of Sale;
+* **PoS - Create Order Line**: The cashier can create the order line of a product in Point Of Sale;
+* **PoS - Change Cashier**: The cashier can select another cashier in Point Of Sale;
+* **PoS - Change Customer**: The cashier can select a customer in Point Of Sale;
```

Comment:
it's mostly for Multi-Pos Systems

Reply:
Actually not only. It can be useful when customer is selected by scanning his barcode. So, it could be renamed to ``Manual Customer Selecting``